scripts/pre_commit_linter.py

```diff
@@ -1138,10 +1331,77 @@ def _lint_py_files(
         result.put('%s   %s Python files linted (%.1f secs)' % (
             _MESSAGE_TYPE_SUCCESS, num_py_files, time.time() - start_time))
 
-    print 'Python linting finished.'
+    print('Python linting finished.')
+
 
+def _lint_py_files_for_python3_compatibility(
+        files_to_lint, result, verbose_mode_enabled):
+    """Prints a list of Python 3 compatibility errors in the given list of
+    Python files.
 
-class LintChecksManager:
+    Args:
+        files_to_lint: list(str). A list of filepaths to lint.
+        result: multiprocessing.Queue. A queue to put results of test.
+        verbose_mode_enabled: bool. True if verbose mode is enabled.
+    """
+    start_time = time.time()
+    are_there_errors = False
```

Comment:
Let's make this an int, `errors_found`, to print the exact count.

Reply:
We are just extracting the output from pylint and if there is any error(i.e if there is any output generated by pylint), we are setting are_there_errors to be True so that the appropriate msg can be printed. I don't think we can count the number of errors as pylint does that internally. We are just printing the output by pylint

Reply:
I see. Let's just name it `any_errors` then.

Reply:
Done